runtime/src/main/java/org/corfudb/runtime/view/CacheOption.java

```diff
@@ -8,12 +8,11 @@
  */
 public enum CacheOption {
     /**
-     * Issue a write without caching the result
+     * Don't cache
      */
-    WRITE_AROUND,
+    NO_CACHE,
     /**
-     * Issue a write and cache the result if the write
-     * succeeds
+     * Write then cache
      */
-    WRITE_THROUGH
+    WRITE_BACK
```

Comment:
Just curious, we don't do eventual persistence right, so why is it WRITE_BACK instead of WRITE_THROUGH?

Reply:
It should be write through, I don't remember changing this. Will remove that commit.